cluster-configuration/services-configuration.yaml

```diff
@@ -109,6 +109,13 @@ prometheus:
   prometheus-port: 9091
   # port for node exporter
   node-exporter-port: 9100
+  alerting:
+    alert_manager_port: 9093
+    alert_receiver: alert@example.com
```

Comment:
suggest Comment out the default config as an example. User maybe use quick config and not aware this config. The fake mail may cause error